include/version.h.in

```diff
@@ -0,0 +1,3 @@
+#pragma once
```

Comment:
For consistency with the other header files I think we should keep using the standard include guard system here instead of the pragma. (also, I think `#pragma` is technically C99 and we want files that could be included in the code generation to be C89 for now).

Reply:
Done